### PR TITLE
Ignore failed SNMP value get for GroupCheck

### PR DESCRIPTION
**Pull request by kfrukacz-gs**
When flag is set to ignore on a GroupCheck it now actually allows to ignore a failed SNMP values get. This means that if the values are not fetched the error is ignored and the value is 0.
Use case: trigger heal workflow when cluster stops sending traffic
 - SNMP values are fetched when nodes are running allowing for legacy triggers
 -  when all nodes in cluster fail to return value to SNMP GET and the on_unknown is set to ignore the value is 0
 - low trigger threshold set to 0 allows to execute heal (or other workflow)

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,4 @@
+1.0.7 - Add igore flag processing to check_group_aggregate and check_group_meta_aggregate
```
Comment (EarthmanT) [inline CHANGELOG]: ignore

File: examples/blueprints/dummygroup.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.3.dev1/types.yaml
```
Comment (EarthmanT) [inline examples/blueprints/dummygroup.yaml]: can we use `cloudify.co` and `5.1`?

File: examples/blueprints/dummygroup.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  - http://www.getcloudify.org/spec/openstack-plugin/2.0.1/plugin.yaml
```
Comment (EarthmanT) [inline examples/blueprints/dummygroup.yaml]: Can we use 2.14.22.

**Review by EarthmanT — CHANGES_REQUESTED** [review]

File: examples/blueprints/dummygroup.yaml (hunk 3 above)
Comment (kfrukacz-gs) [reply examples/blueprints/dummygroup.yaml]: getting access denied for http://cloudify.co/spec/openstack-plugin/2.14.22/plugin.yaml

File: CHANGELOG (hunk 1 above)
Comment (kfrukacz-gs) [reply CHANGELOG]: done

File: examples/blueprints/dummygroup.yaml (hunk 2 above)
Comment (kfrukacz-gs) [reply examples/blueprints/dummygroup.yaml]: done

**Pull request merged**